package.json

```diff
@@ -14,7 +14,6 @@
     "prettier": "^1.11.1"
   },
   "scripts": {
-    "preseed": "yarn install",
```

Comment:
Why remove this?

Reply:
I was mistaken about something--I'll revert this